### ISSUE TITLE
Is the released `pvsm_finetuned_full.pt` the final model?

### ISSUE THREAD

**Issue by jk4011** [wuzirui/pvsm#4]
Hi, thanks for releasing the code and checkpoints!

With the released 24-layer checkpoint I get lower numbers than Table 2 on the RE10K overlap benchmark (same index file, 2 context + 3 target views, 256×256, settings from `configs/runs/pvsm_finetuned_full.yaml`, all weights loaded):

| | Small | Medium | Large | Total (PSNR) |
|---|---|---|---|---|
| Paper, Table 2 (24 layers) | 24.98 | 26.88 | 28.60 | 26.90 |
| Mine, RE10K poses + MapAnything depth | 23.41 | 25.80 | 27.67 | 25.76 |

While debugging I noticed that the checkpoint seems to be saved at step 500 of a run, with the learning rate still in warm-up:

```python
ck = torch.load("pvsm_finetuned_full.pt", map_location="cpu", weights_only=False)
ck["param_update_step"], ck["lr_scheduler"]["_last_lr"]   # 500, [6.67e-05, 6.67e-05]
```

Is this the final model used for Table 2? 
If it is an early snapshot of the last fine-tuning stage, could you release the final checkpoint?

**Comment by wuzirui** [comment]
Hi! The released 24-layer checkpoint includes an additional 500 steps of camera-augmentation fine-tuning. I’ll document the training stage of each checkpoint more explicitly.

The performance issue might originate from the depth data, we have some preprocessing steps regarding the raw MapAnything depth. I've attached the scripts here for your reference.

Let me know if this fixes your issue. :)
[process_re10k_with_mapanything.zip](https://github.com/user-attachments/files/32566838/process_re10k_with_mapanything.zip)
